project-management/src/test/groovy/life/qbic/projectmanagement/application/ProjectCreationServiceSpec.groovy

```diff
@@ -47,7 +47,7 @@ class ProjectCreationServiceSpec extends Specification {
     given:
     projectRepositoryStub.add(_ as Project) >> {}
     addExperimentToProjectServiceStub.addExperimentToProject(_ as ProjectId, _ as String as String, _ as List<Species> as List<Species>, _ as List<Specimen> as List<Specimen>, _ as List<Analyte>) >> {}
-    def contact = new Contact("Mustermann", "some@notavailable.zxü")
+    def contact = new Contact("Mustermann", "some@notavailable.zxü", "", "")
```

Comment:
The issue with empty values vs non-existing values. (See my comments in the Contact class)
Think about providing different means of object creation.